web/pandas/versions.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "1.5 (dev)",
+        "version": "docs/dev"
+    },
+    {
+        "name": "1.4.0 (rc)",
+        "version": "pandas-docs/version/1.4"
+    },
+    {
+        "name": "1.3.5 (stable)",
+        "version": "docs"
+    },
+    {
+        "name": "1.3.5",
+        "version": "pandas-docs/version/1.3"
```

Comment:
Question: is there any preference in showing the fully qualified version ("1.3.5"), or rather a simplified one ("1.3") ? 
Also referring to django (https://docs.djangoproject.com/), for example they show the simplified version

Reply:
i would just show the minor version e.g. 1.3